include/cell/compute/base.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
```

Comment:
how about renaming this file into "math_functor.hpp". `base.hpp` seems too general a name.

Reply:
fixed.